pkg/multus/multus.go

```diff
@@ -896,5 +897,22 @@ func CmdDel(args *skel.CmdArgs, exec invoke.Exec, kubeClient *k8s.ClientInfo) er
 		}
 	}
 
-	return delPlugins(exec, pod, args, k8sArgs, in.Delegates, len(in.Delegates)-1, in.RuntimeConfig, in)
+	e := delPlugins(exec, pod, args, k8sArgs, in.Delegates, len(in.Delegates)-1, in.RuntimeConfig, in)
+
+	// delegate plugin delete success, delete cache file
+	// others
+	// 1.  sandBox container stop times, but delegated plugin failed (cache file has been deleted)
+	// 2.  container gc manager clean up forever even though POD has been fully deleted from ETCD
+	// this case POD not find && cache file not find; never clean up
```

Comment:
I'll make a suggestion for a comment update as well (todo)

Reply:
update something ....